text/0097-log-data-model.md

```diff
@@ -0,0 +1,972 @@
+    <td>The request method.</td>
+    <td>attributes["http.method"]</td>
+  </tr>
+  <tr>
+    <td>%v,%p,%U,%q</td>
+    <td>string</td>
+    <td>Multiple fields that can be composed into URL.</td>
+    <td>attributes["http.url"]</td>
+  </tr>
+  <tr>
+    <td>%>s</td>
+    <td>string</td>
+    <td>Response status.</td>
+    <td>attributes["http.status_code"]</td>
+  </tr>
+  <tr>
+    <td>All other fields</td>
+    <td>any</td>
+    <td>Structured data.</td>
+    <td>attributes, key=TBD</td>
+  </tr>
+</table>
+
+
+### CloudTrail Log Event
+
+<table>
+  <tr>
+    <td>Field</td>
+    <td>Type</td>
+    <td>Description</td>
+    <td>Maps to Unified Model Field</td>
+  </tr>
+  <tr>
+    <td>eventTime</td>
+    <td>string</td>
+    <td>The date and time the request was made, in coordinated universal time (UTC).</td>
+    <td>timestamp</td>
+  </tr>
+  <tr>
+    <td>eventSource</td>
+    <td>string</td>
+    <td>The service that the request was made to. This name is typically a short form of the service name without spaces plus .amazonaws.com.</td>
+    <td>resource["service.name"]?</td>
+  </tr>
+  <tr>
+    <td>awsRegion</td>
+    <td>string</td>
+    <td>The AWS region that the request was made to, such as us-east-2.</td>
+    <td>resource["cloud.region"]</td>
+  </tr>
+  <tr>
+    <td>sourceIPAddress</td>
+    <td>string</td>
+    <td>The IP address that the request was made from.</td>
+    <td>resource["net.peer.ip"] or resource["net.host.ip"]? TBD</td>
+  </tr>
+  <tr>
+    <td>errorCode</td>
+    <td>string</td>
+    <td>The AWS service error if the request returns an error.</td>
+    <td>short_description</td>
+  </tr>
+  <tr>
+    <td>errorMessage</td>
+    <td>string</td>
+    <td>If the request returns an error, the description of the error.</td>
+    <td>body</td>
+  </tr>
+  <tr>
+    <td>All other fields</td>
+    <td>*</td>
+    <td></td>
+    <td>attributes["cloudtrail.*"]</td>
+  </tr>
+</table>
+
+
+## Appendix B: `SeverityNumber` example mappings.
+
+|Syslog       |WinEvtLog  |Log4j |Zap   |java.util.logging|SeverityNumber|
+|-------------|-----------|------|------|-----------------|--------------|
+|             |           |TRACE |      | TRACE           |TRACE         |
+|Debug        |Verbose    |DEBUG |Debug | FINER           |DEBUG         |
+|             |           |      |      | FINE            |DEBUG2        |
+|             |           |      |      | CONFIG          |DEBUG3        |
+|Informational|Information|INFO  |Info  | INFO            |INFO          |
+|Notice       |           |      |      |                 |INFO2         |
+|Warning      |Warning    |WARN  |Warn  | WARNING         |WARN          |
+|Error        |Error      |ERROR |Error | SEVERE          |ERROR         |
+|Critical     |Critical   |      |Dpanic|                 |ERROR2        |
+|             |           |      |Panic |                 |ERROR3        |
+|Alert        |           |FATAL |Fatal |                 |FATAL         |
```

Comment:
There is missing `emergency` level from syslog.

Reply:
Good catch. Added.